mappings/net/minecraft/world/World.mapping

```diff
@@ -282,15 +334,22 @@ CLASS net/minecraft/class_99 net/minecraft/world/World
 	METHOD method_357 getLightAtPos (Lnet/minecraft/class_106;Lnet/minecraft/class_1372;)I
 		ARG 1 type
 		ARG 2 pos
+	METHOD method_358 addUnloadedBlockEntity (Lnet/minecraft/class_348;)V
```

Comment:
this should probably be `queueBlockEntity`, since the list is cleared and block entities are added in the next tick.